cmd/list.go

```diff
@@ -289,3 +310,36 @@ func getOptionalIntField(value int64) string {
 	}
 	return fmt.Sprintf("%d", value)
 }
+
+func sortListData(data [][]any, header []string, sortColumn string) {
+	sortColumn = strings.ToLower(sortColumn)
+	if !slices.Contains(header, sortColumn) {
+		fmt.Println("Warning: column '" + sortColumn + "' does not exist")
+		return
+	}
+
+	sortColumnId := slices.Index(header, sortColumn)
+	sort.SliceStable(data, func(i, j int) bool {
+		value1 := data[i][sortColumnId]
+		value2 := data[j][sortColumnId]
+
+		// Make sure missing values are at the bottom of the list
+		if value1 == "-" {
+			return false
+		}
+		if value2 == "-" {
+			return true
+		}
+
+		switch value1.(type) {
+		case int64:
+			return value1.(int64) > value2.(int64)
+		case float64:
+			return value1.(float64) > value2.(float64)
+		case string:
+			return value1.(string) > value2.(string)
+		default:
+			return false // Keep order if any other type
+		}
```

Comment:
I tried to look for a general way to convert the values into an ordered interface but I don't think GO allows that. So I just included in the switch case the types used in the list command. Should I include others? (there are a lot of them, e.g., int, int32, float32, uint8, etc.)

Reply:
I could also do the fmt.Sprint() for the default value, instead of keeping the order

Reply:
Sounds good. For now, I'll just print the error and exit. This can also be improved in #16